admin_light_user/models/ir_ui_view.py

```diff
@@ -0,0 +1,37 @@
+# © 2024 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+
+
+class View(models.Model):
+    _inherit = "ir.ui.view"
+
+    def get_inheriting_views_arch(self, model):
```

Comment:
We don't need to change in the inheriting views. we can change directely in the method  "update_admin_light_user_group_view()" to ckeck if the user hase group "admin light user" and not group "group_erp_manager" before updating the content of the form view